test/types/findByPk.ts

```diff
@@ -1,3 +1,34 @@
+import { expectTypeOf } from 'expect-type'
+import { Attributes } from '@sequelize/core';
 import { User } from './models/User';
 
-User.findByPk(Buffer.from('asdf'));
+async () => {
```

Comment:
same note about making this a IIFE so linters don't error on this when we enable them on these files :)

As well as on `findOne.ts`